cypress/integration/external/sharing-access-with-existing-user.spec.js

```diff
@@ -0,0 +1,50 @@
+const { setUp, tearDown } = require('../../support/setup');
+
+describe('External user sharing license access with another external user ', () => {
+  beforeEach(() => {
+    tearDown();
+    setUp('barebones');
+  });
+
+  after(() => {
+    tearDown();
+  });
+
+  it('User login and assing the license to another external user', () => {
+    //  User logs in
+    cy.visit(Cypress.env('USER_URI'));
+    cy.get('a[href*="/signin"]').click();
+    cy.fixture('users.json').then(users => {
+      cy.get('input#email').type(users.external);
+    });
+    cy.get('input#password').type(Cypress.env('DEFAULT_PASSWORD'));
+    cy.get('.govuk-button.govuk-button--start').click();
+
+    //  view and assign the licence
+    cy.get('.licence-result__column > a').contains('AT/CURR/DAILY/01').should('be.visible');
+    cy.get('#navbar-manage').contains('Add licences or give access').click();
+    cy.get('.govuk-list').contains('Give or remove access to your licence information').click();
+    cy.get('.govuk-grid-column-two-thirds > .govuk-button').click();
```

Comment:
Here on line 27 you are picking up on the button by its classname. Instead, I think you ought to identify it based on its text, or by URL. This is so that if the position of the link changes in the future, the test shouldn't necessarily fail.